app/publishing/delinkedCalculation/update-published.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../../data')
+
+const updatePublished = async (calculationReference, transaction) => {
+  await db.delinkedCalculation.update({ datePublished: new Date() }, { where: { calculationId: calculationReference }, transaction })
```

Comment:
We should just pass the variable as calculationId, not calculationReference. We can then shorthand the update function:

`await db.delinkedCalculation.update({ datePublished: new Date() }, { where: { calculationId }, transaction })`